toolkits/global/packages/global-status-message/view/globalStatusMessage.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="c-status-message{{#if boxed}} c-status-message--boxed{{/if}}{{#if type}} c-status-message--{{type}}{{/if}}"{{#if liveRegion}} role="status" aria-live="polite"{{/if}}>
```

Comment:
Shall we have a `tagContainer` prop that would care for using `aside` element for semantic and accessibility sake. FYI, here is our current implementation of the status message partial:
https://github.com/springernature/researcher-portal/blob/master/researcher-portal-frontend/src/view/partials/status-message.hbs

Reply:
Ah, I forgot to make it an aside. I don't think we need an option, I have now set it as an `<aside>` as standard.